### PR TITLE
Add catalan language

### PR DESCRIPTION
**Pull request by ghost**
Could be nice to add Catalan language to Mastodon !

### CODE REVIEW

File: app/javascript/mastodon/containers/mastodon.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ import de from 'react-intl/locale-data/de';
 import en from 'react-intl/locale-data/en';
 import eo from 'react-intl/locale-data/eo';
 import es from 'react-intl/locale-data/es';
+import ca from 'react-intl/locale-data/ca';
```
Comment (ykzts) [inline app/javascript/mastodon/containers/mastodon.js]: sort by name

File: app/javascript/mastodon/containers/mastodon.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ addLocaleData([
   ...en,
   ...eo,
   ...es,
+  ...ca,
```
Comment (ykzts) [inline app/javascript/mastodon/containers/mastodon.js]: sort by name

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/locales/ca.yml [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+      resolved: Resolt
+      silence_account: Silenciar compte
+      status: Estat
+      suspend_account: Suspendre compte
+      target: Objectiu
+      title: Informes
+      unresolved: No resolt
+      view: Vista
+    settings:
+      click_to_edit: Clic per editar
```
Comment (ykzts) [inline config/locales/ca.yml]: It is an unused key.  Remove this line.

Comment (salvadorpla) [inline config/locales/ca.yml]: Sorry, what line ? Thank you

Comment (ykzts) [inline config/locales/ca.yml]: ? this line. please see GitHub web UI.

**Review by ghost — COMMENTED** [review]
Correction done.

**Review by ghost — COMMENTED** [review]
Sorted by name

**Review by ghost — COMMENTED** [review]
Done

**Review by ghost — COMMENTED** [review]
Done

**Review by ykzts — APPROVED** [review]

**Pull request merged**